test/test_dataframe.py

```diff
@@ -79,15 +79,15 @@ def test_dataframe_maker_iterdatapipe(self):
         # Functional Test: DataPipe correctly converts into a single TorchArrow DataFrame
         df_dp = source_dp.dataframe(dtype=DTYPE)
         df = list(df_dp)[0]
-        expected_df = torcharrow.DataFrame([(i,) for i in range(10)], dtype=DTYPE)
+        expected_df = torcharrow.dataframe([(i,) for i in range(10)], dtype=DTYPE)
```

Comment:
curious: why the test failure doesn't trigger before?

Reply:
We don't have a OSS CI setup for `torcharrow`, I'm looking into setting one up right now.